polxml

```diff
@@ -366,6 +366,126 @@ class Topology(object):
+                        outputfile.write(' {:d} Drude particles //\n\n'.format(self.ndrude))
+                        del data[line:line+3]
+
+                    if data[line] == "field = 'field.xml'\n":
+                        outputfile.write("field = 'field-p.xml'\n")
+                        count = count+1
+                    
+                    if data[line] == "config = 'config.pdb'\n":
+                        if '.mmcif' in config_input:
+                            outputfile.write("config = 'config-p.mmcif'\n")
+                            count = count+1
+                        else :
+                            outputfile.write("config = 'config-p.pdb'\n")
+                            count = count+1
+                    
+                    if data[line] == "config = 'config.mmcif'\n":
+                        outputfile.write("config = 'config-p.mmcif'\n")
+                        count = count+1
+
+                    if data[line] == "pdb = app.PDBFile(config)\n":
+                        if '.mmcif' in config_input:
+                            outputfile.write("#pdb = app.PDBFile(config)\n")
+                            outputfile.write("# If PDBx/mmCIF format is used as config file:\n")
+                            outputfile.write("pdb = app.PDBxFile(config)\n\n")
+                            del data[line:line+3]
+                            count = count+1
+                    
+                    if data[line] == "### Integrator selection ###\n":
+                        del data[line:line+5]
+                        outputfile.write("### Integrator selection ###\n")
+                        outputfile.write("#print('# Nose-Hoover integrator')\n")
+                        outputfile.write("#integrator = openmm.NoseHooverIntegrator(temperature, 5/unit.picosecond, 1*unit.femtosecond)\n")
+                        outputfile.write("print('Drude Nose-Hoover integrator', temperature)\n")
+                        outputfile.write("integrator = openmm.DrudeNoseHooverIntegrator(temperature, 5/unit.picosecond,\n")
+                        outputfile.write("    1*unit.kelvin, 1/unit.picosecond, 1*unit.femtosecond)\n")
+                        outputfile.write("#print('# Drude Langevin integrator')\n")
+                        outputfile.write("#integrator = openmm.DrudeLangevinIntegrator(temperature, 5/unit.picosecond,\n")
+                        outputfile.write("#    1*unit.kelvin, 20/unit.picosecond, 1*unit.femtosecond)\n")
+                        outputfile.write("integrator.setMaxDrudeDistance(0.2*unit.angstrom)\n")
+                        outputfile.write("print('#   max Drude distance', integrator.getMaxDrudeDistance())\n\n")
+                        count = count+1
+
+                    if data[line] == "### Run simulation ###\n":
+                        del data[line:line+4]
+                        outputfile.write("### Constants ###\n")
```

Comment:
As in fftool, most of this is immutable text, so more suitable in a """..."""
Also, the question if this is the best way. Maybe include an omm-p.py example with polxml?